web/themes/webspark/renovation/includes/theme.inc

```diff
@@ -146,3 +147,89 @@ function renovation_preprocess_media(array &$variables) {
     unset($attributes['data-round']);
   }
 }
+
+
+/**
+ * Implements hook_preprocess_maintenance_page().
+ */
+
+function renovation_preprocess_maintenance_page(&$variables) {
+
+  $block_manager = \Drupal::service('plugin.manager.block');
+  
+  // Building header block and props, no navtree
+
+  $header_block = \Drupal\block\Entity\Block::load('asubrandheader');
+
+  // AsuBrandHeaderBlock
+
+  if ($header_block) {
+    $settings = $header_block->get('settings');
+    $header_built = $block_manager->createInstance('asubrandheader', $settings);
+    // Some blocks might implement access check.
+    $header_render =  $header_built->build();
+    // Add the cache tags/contexts.
+    \Drupal::service('renderer')->addCacheableDependency($header_render, $header_built);  
+
+    $header_config = $header_block->get('settings');
+
+    // why do I need to rebuild this here? I can't figure out a way to access the props object created in modules/webspark/asu_brand/src/Plugin/Block/AsuBrandHeaderBlock.php directly. Doing it here does give me the opportunity to set navtree to null though.
+    $props = [];
+    $props['baseUrl'] = $header_config['asu_brand_header_block_base_url'] ?? 'https://asu.edu';
+    $props['title'] = $header_config['asu_brand_header_block_title'];
+    $props['parentOrg'] = $header_config['asu_brand_header_block_parent_org'];
+    $props['parentOrgUrl'] = $header_config['asu_brand_header_block_parent_org_url'];
+    $props['expandOnHover'] = $header_config['asu_brand_header_block_expand_on_hover'];
+    $props['loginLink'] = $header_config['asu_brand_header_block_login_path'];
+    $props['logoutLink'] = $header_config['asu_brand_header_block_logout_path'];
+    $props['searchUrl'] = "https://search.asu.edu/search";
```

Comment:
I'm okay with the `searchUrl` being hardcoded, but there's actually a sitewide config for that value and the `site` prop as well, as seen here: https://github.com/ASUWebPlatforms/webspark-ci/blob/master/web/modules/webspark/asu_brand/src/Plugin/Block/AsuBrandHeaderBlock.php#L117-L123

If you were to implement the props helper function, that would be replicated within there.  Otherwise, I'm okay with this being hardcoded in maintenance mode, but it becomes one of those things that's "hidden away" and may get overlooked in future updates.